doc/guides/4 - Refinery Extensions/4 - Pretty URLs.textile

```diff
@@ -8,30 +8,31 @@ This guide will show you how to:
 
 endprologue.
 
-WARNING: This method is tested with Refinery CMS v. 2.0.0 only. It should be applicable to later versions, but not earlier versions (1.0.9 or earlier). Always ensure your code is backed up or under version control before making substantial change.
+WARNING: This method is tested with Refinery CMS v. 2.0.0 only. It should be applicable to later versions, but not earlier versions (1.0.9 or earlier). It also works with v 3.0, although it has not been tested extensively.
+Always ensure your code is backed up or under version control before making substantial changes.
 
 h3. Fresh Slugs
 
 FriendlyID requires that you create a string field entitled +slug+ on any model that you wish to convert. It's better to do this as a separate migration after creating the extension, because you don't want Refinery to auto-generate a field in the back-end; this attribute is set by code, not by hand.
 
-To do so, create a new migration file in the +db/migrate+ folder of your extension. Since this depends on the previous migration, it's best to rename the other migrations to include a sequence reference in the filename -- for example, +create_staff_members.rb+ becomes +001_create_staff_members.rb+. When you re-run the generator that will move these migrations over to your main +db/migrate+ folder, they will be moved in this order. Don't worry if you've already run these migrations; Rails will auto-detect that you've run them, even though the filename has been changed, and it won't re-run them.
```

Comment:
I prefer this older instruction.  It's a good idea to place the migration inside the extension and then using `rails g refinery:staff_members` to copy it to the application.  This ensures that if you gemify the extension that it has everything that it needs right out of the box.  Otherwise, if you copy this to another application, it will be missing some of its migrations.